client/src/app/auth/offline-auth.service.ts

```diff
@@ -1,3 +1,5 @@
+import * as firebase from 'firebase';
```

Comment:
This should also go in a separate commit for fixing TS.

Still dunno why this didn't come up earlier.